addons/common/functions/fnc_findUnloadPosition.sqf

```diff
@@ -40,12 +40,22 @@ if (_checkVehicleIsStable) then {
 };
 
 private _radiusOfItem = 1;
-if (_typeOfCargo isKindOf "CAManBase") then {
+if (_cargo isKindOf "CAManBase") then {
     _radiusOfItem = 1.1;
 } else {
     //`sizeOf` is unreliable, and does not work with object types that don't exist on map, so estimate size based on cargo size
-    if (isNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) then {
-        _radiusOfItem = (((getNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) ^ 0.35) max 0.75);
+    private _typeOfCargo = if (_cargo isEqualType "") then {_cargo} else {typeOf _cargo};
+    private _itemSize = if (isNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size))) then {
+        getNumber (configFile >> "CfgVehicles" >> _typeOfCargo >> QEGVAR(cargo,size));
+    } else {
+        if (["ace_cargo"] call EFUNC(common,isModLoaded)) then {
+            [_cargo] call EFUNC(cargo,getSizeItem);
+        } else {
+            _radiusOfItem;
+        };
+    };
+    if !(_itemSize isEqualTo -1) then {
```

Comment:
just `_itemSize != -1`

Reply:
With `isEqualTo` you can compare  different type like: `nil isEqualTo -1` so the function will never fail
> It doesn't throw error when comparing different types, i.e. ("eleven" isEqualTo 11)
But, if ace use it as convention I will change it.

Reply:
`_itemSize` here is always number